src/main/java/org/frankframework/frankdoc/model/ElementType.java

```diff
@@ -104,14 +104,14 @@ private static String parseDefaultElementTag(FrankClass clazz, FrankClassReposit
 		if(StringUtils.isBlank(value)) {
 			// null means the JavaDoc tag was not present - then nothing to do.
 			if(value != null) {
-				log.warn("JavaDoc tag {} of interface [{}] should have a parameter", JAVADOC_DEFAULT_CLASSNAME, clazz.getName());
+				log.error("JavaDoc tag {} of interface [{}] should have a parameter", JAVADOC_DEFAULT_CLASSNAME, clazz.getName());
 			}
 		} else {
 			try {
 				FrankClass defaultClass = repository.findClass(value);
 				result = defaultClass.getName();
 			} catch(FrankDocException e) {
-				log.warn("JavaDoc tag {} on interface [{}] does not point to a valid class: [{}]", JAVADOC_DEFAULT_CLASSNAME, clazz.getName(), value);
+				log.error("JavaDoc tag {} on interface [{}] does not point to a valid class: [{}]", JAVADOC_DEFAULT_CLASSNAME, clazz.getName(), value);
```

Comment:
Hier moet je ook de exceptie loggen

Reply:
Deze warning / error betekent dat er een ff.defaultElement is gezet die verwijst naar een niet-bestaande class. Dit heeft hetzelfde effect als wanneer je het ff.defaultElement tag weg laat. Als we hier weer een warning van maken, dan laten we onheldere code toe die gemakkelijk valt te fixen. Wil je dat echt?

Reply:
Gerrit bedoeld hier om ook de exceptie (`e`) mee te geven aan de logger. Nu vang je alles af en ga je er altijd van uit dat de fout altijd verwijst naar een niet-bestaande class.

Reply:
Gedaan.